packages/style-engine/src/styles/color/background.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import type { Style, StyleOptions } from '../../types';
+import { generateRule, getSlugFromPreset } from '../utils';
+
+const text = {
```

Comment:
Tiny typo: looks like this should be `background`.

Reply:
🤦